index.md

```diff
@@ -9,14 +9,13 @@ subnav-link-gradient: "blueberry-lime-link"
 nav_link: home
 id: home
 sections:
-  - Overview
```

Comment:
Why was this removed?

Reply:
Umm It shouldn't have been? I did a simple find and then replaced, so not sure how that happened. Will change.

Reply:
This section is still removed in latest push.  Need to make sure this gets added back in.